tests/test_project.py

```diff
@@ -533,16 +532,16 @@ def do_run(self, args, ignored):
                               print('Testing kconfig test command')
                       '''),
                       })
-    west_tmpdir = repos_tmpdir.join('west_installation')
-    cmd('init -m "{}" "{}"'.format(str(repos_tmpdir.join('repos', 'zephyr')),
-                                   str(west_tmpdir)))
+    west_tmpdir = repos_tmpdir / 'west_installation'
```

Comment:
is `repos_tmpdir` created with pathlib? or does this `/` work with standard `os.path` objects or even with strings?

Reply:
This repos_tmpdir is an os.PathLike object that ultimately is derived from a pytest 'tmpdir' fixture object, which is a pytest LocalPath instance. I'm not sure, but my guess is that the design of LocalPath predates and influenced the standard library pathlib.Path types, since they work in very similar ways.